City-of-Helsinki/berth-reservations-admin
# VEN-1093 | Add Application Code Filter

## Description :sparkles:
- Adding a new filter for the application list 
- Add the ability for the filters to be combined

## Issues :bug:

### Closes :no_good_woman:
[VEN-1093](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1093): Filter applications by ´application code exists´

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️
- Updated relevant snapshots

### Manual testing :construction_worker_man:
- Go to the application list view by clicking `Venepaikkahakemukset` on the sidebar
- From the table's header, you can click on `HAKUKOODI` to display only the applications that have codes (new and exchange applications)
- You should be able to combine it with other filters, e.g: `HAKUKOODI` + `VAIHTOHAKEMUKSET`
- By clicking on the filter again, it disables the filter

## Screenshots :camera_flash:
![image](https://user-images.githubusercontent.com/23040926/110789081-29913280-8278-11eb-83bd-a1ad8050a41d.png)

## Additional notes :spiral_notepad:
